govcd/org.go

```diff
@@ -428,9 +365,8 @@ func (adminOrg *AdminOrg) Disable() error {
 		return fmt.Errorf("error getting AdminOrg HREF %s : %v", adminOrg.AdminOrg.HREF, err)
 	}
 	orgHREF.Path += "/action/disable"
-	req := adminOrg.client.NewRequest(map[string]string{}, "POST", *orgHREF, nil)
-	_, err = checkResp(adminOrg.client.Http.Do(req))
-	return err
+
+	return adminOrg.client.ExecuteRequestWithoutResponse(orgHREF.String(), http.MethodPost, "", "", nil)
```

Comment:
Please add a meaningful error message to this call, or it will fail for lack of placeholder.

Reply:
fixed